README.adoc

```diff
@@ -244,21 +236,21 @@ status/microprofile-config.properties
 include::finish/status/src/main/resources/META-INF/microprofile-config.properties[]
 ----
 
-The `Status` microservice receives the message from the `Kitchen`, `Bar`, `Order`, and `ServingWindow` microservices over the [hotspot=updateStatus file=3]`@Incoming("updateStatus")` channel with its properties defined in the [hotspot=updateStatus file=4]`microprofile-config.properties`.
+The `Status` microservice receives the message from the `Bar`, `Kitchen`, `Order`, and `ServingWindow` microservices over the [hotspot=updateStatus file=0]`@Incoming("updateStatus")` channel with its properties defined in the [hotspot=updateStatus file=1]`microprofile-config.properties`.
```

Comment:
At top the order is Order Kitchen bar. I got a feedback to keep the same order throughout the guide.
Each of the five microservices `Order`, `Kitchen`, `Bar`, `ServingWindow`, and `Status` contain `microprofile-config.properties`

Reply:
I visited and updated the whole guide. The order should be alphabetic: Bar, Kitchen, OpenLibertyCafe, Order, ServingWindow, and Status